docs/configuration/configuration.md

```diff
@@ -451,6 +451,26 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Lower limit for the growth factor of one bucket to the next in each native histogram. 
+# The resolution of a histogram with a lower growth factor will be reduced until it is within the limit.
+# Here is the conversion table from growth factor to resulting schema/scale.
+# | growth factor | resulting schema AKA scale |
+# | ---   | --- |
+# | 65536 | -4  |
+# | 256   | -3  |
+# | 16    | -2  |
+# | 4     | -1  |
+# | 2     |  0  |
+# | 1.4   |  1  |
+# | 1.1   |  2  |
+# | 1.09  |  3  |
+# | 1.04  |  4  |
+# | 1.02  |  5  |
+# | 1.01  |  6  |
+# | 1.005 |  7  |
+# | 1.002 |  8  |
+[ native_histogram_min_bucket_factor: <float> | default = 1.00271 ]
```

Comment:
This default, while it makes sense, looks very obscure to those that do not know the internals of native histograms. Could we be consistent and just use a default of 0, with the usual premise that "0 means no limit"?

Reply:
I agreed with this expression. 
Just to make sure, the 'no limit' should mean the schema should still not be larger than 8, rather than being unlimitedly large, right?
@beorn7

Reply:
Currently, it means schema 8, because that is the largest schema Prometheus currently supports. But we might decide to support larger schemas in the future. So 0 essentially means "lowest factor supported by Prometheus, currently that results in 1.00271".